cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestManager.java

```diff
@@ -334,10 +337,35 @@ public void deleteNamespace(Id.Namespace namespace) throws Exception {
   }
 
   @Override
+  public void deleteNamespaceData(NamespaceId namespace) throws Exception {
```

Comment:
I'm going to deprecate the ``createNamespace`` and ``deleteNamespace`` methods and instead expose the ``NamespaceAdmin`` via a static getter, just like ``IntegrationTestBase``.